Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+#if AXIS
+
+@objc
+protocol NYPLBookDownloadBroadcasting {
+  
+  /// Shows an alert notifying the user about book download failure.
+  /// - Parameter book: The book associated with download failure.
+  func failDownloadWithAlert(forBook book: NYPLBook)
+  
+  /// Moves the downloaded book file from current location to desired location.
+  /// - Parameters:
+  ///   - book: The book that finished downloading.
+  ///   - sourceLocation: Current location of book file(s).
+  ///   - downloadtask: Downlaod task for book download.
+  func replaceBook(_ book: NYPLBook,
+                   withFileAtURL sourceLocation: URL,
+                   forDownloadTask downloadtask: URLSessionDownloadTask) -> Bool
+}
+
+private struct AxisHelper {
+  static let isbnKey = "isbn"
+  static let bookVaultKey = "book_vault_uuid"
+  /// This is the url for downloading content for a given book with axis drm.
+  static let baseURL = URL(string: "https://node.axisnow.com/content/stream/")!
+}
+
+@objc
+class AxisService: NSObject {
```

Comment:
can you rename this to `NYPLAxisService`? feel free to do in later PR if easier

Reply:
Done